test/fixtures/retail_customer_data.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@dfcoffin, could you help me to verify this sample customer data with Green Button standard at your available time?

Thanks.